src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+
+const averageAge = (array) => array
+  .reduce((accumulator, value) =>
+    accumulator + (value.died - value.born), 0) / array.length;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const onlyMen = people.filter(person =>
+    century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm'
+  );
+
+  return +(averageAge(onlyMen)).toFixed(2);
```

Comment:
.toFixed() also repeated several times, you can move it inside `averageAge` funtion to call just this function `return averageAge(onlyMen)`